Utilities/HttpServer.cs

```diff
@@ -98,6 +101,58 @@ public class HttpServer {
       }
     }
 
+    public static IDictionary<string, string> ToDictionary(NameValueCollection col) {
+      IDictionary<string, string> dict = new Dictionary<string, string>();
+      foreach (var k in col.AllKeys) {
+        dict.Add(k, col[k]);
+      }
+      return dict;
+    }
+
+    public SensorNode findSensor(Node node, string id) {

```

Comment:
Can you change this to C# formatting (i.e., FindSensor)?

Reply:
Done.